evaluation.md

```diff
@@ -5,6 +5,31 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+First, we added a package line and put all the code in an object, because we are
+pretty sure they are mandatory. We got rid of the curly braces around the state
+declarations because we realized we could go without them and decided they were
+not the most beautiful. We added "and" between the direction and the next state
```

Comment:
After reading the example I think I prefer `and` to then - to me it makes more conceptual sense.

Maybe make them equivalent functions, if you really wanted to?